.env.test

```
@@ -0,0 +1,31 @@
+# ============================================
+# Redis Configuration
+# ============================================
+REDIS_PASSWORD=change_this_redis_password_in_production
+REDIS_URL=redis://:change_this_redis_password_in_production@127.0.0.1:56379/0
+
+# ============================================
+# Application Configuration
+# ============================================
+NODE_ENV=test
+SESSION_SECRET=change_this_secret_in_production
+ENCRYPTION_KEY=f4ec8639c7a153f6574da72043256652a04f35c9f27f7235ef3de2f792a99371
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ENCRYPTION_KEY key should go before the NODE_ENV key

(UnorderedKey)